### DISCUSSION TITLE
请问面板现在不支持nginx代理吗？

### DISCUSSION THREAD

**Discussion by Thetail001** [jonssonyan/h-ui#88] [Q&A]
nignx里面是这样写的

> 其他的略过，就一般的https配置
location /hui {
    proxy_redirect off;
    proxy_pass https://127.0.0.1:8001;
    proxy_http_version 1.1;
    proxy_set_header Host $host;
   }

访问https://域名/hui显示这个：

> {
    "code": 50401,
    "type": "no",
    "message": "unauthorized",
    "data": null
}

面板里证书也设置了，https://ip加端口还是能正常访问的

**Comment by jonssonyan** [comment]
代理web页面？

**Reply by Thetail001** [reply]
就是想类似x-ui那样可以通过域名加/位置直接访问而不是通过ip地址加端口

**Reply by jonssonyan** [reply]
nginx 配置没问题，问题在/hui，这是后端接口默认的上下文，换一个试试
https://github.com/jonssonyan/h-ui/blob/870f5a4bf045116518465ee9ab0400ef02fa3233/router/router.go#L15C28-L15C31

**Reply by Thetail001** [reply]
把位置换成了/hboard，访问`https://域名/hboard`页面依旧是：

> {
    "code": 50401,
    "type": "no",
    "message": "unauthorized",
    "data": null
}

**Reply by jonssonyan** [reply]
只要代理/
其他的路径不要代理
比如：/assets、/favicon.ico

**Reply by jonssonyan** [reply]
unauthorized 是由于，请求了/hboard/assets 没找到资源导致的

**Reply by Thetail001** [reply]
好的，懂了谢谢

**Reply by jonssonyan** [reply]
上下文会在v0.0.14
